ov_msckf/src/slam2.cpp

```diff
@@ -183,15 +188,16 @@ class slam2 : public plugin {
 		, sb{pb->lookup_impl<switchboard>()}
 		, _m_pose{sb->get_writer<pose_type>("slow_pose")}
 		, _m_imu_integrator_input{sb->get_writer<imu_integrator_input>("imu_integrator_input")}
-		, _m_begin{std::chrono::system_clock::now()}
+		, _m_rtc{pb->lookup_impl<RelativeClock>()}
```

Comment:
Why is this required? It's never used.

Reply:
It's used by the first dummy pose which is then commented out. If we deem the first dummy pose as not required, this is not required either.